lib/storage/metadata/mongoclient/MongoClientInterface.ts

```diff
@@ -411,7 +424,18 @@ class MongoClientInterface {
                 value: {
                     ...newBucketMD,
                     quotaMax: new Long(newBucketMD.quotaMax || '0'),
-                    capabilities: undefined,
+                    capabilities: newBucketMD.capabilities ? {
+                        ...newBucketMD.capabilities,
+                        VeeamSOSApi: newBucketMD.capabilities.VeeamSOSApi ? {
+                            ...newBucketMD.capabilities.VeeamSOSApi,
+                            CapacityInfo: newBucketMD.capabilities.VeeamSOSApi.CapacityInfo ? {
+                                Capacity: new Long(newBucketMD.capabilities.VeeamSOSApi.CapacityInfo.Capacity || '0'),
+                                Available: new Long(newBucketMD.capabilities.VeeamSOSApi.CapacityInfo.Available || '0'),
+                                Used: new Long(newBucketMD.capabilities.VeeamSOSApi.CapacityInfo.Used || '0'),
+                                LastModified: (newBucketMD.capabilities.VeeamSOSApi.CapacityInfo as any).LastModified,
+                            } : undefined,
+                        } : undefined,
```

Comment:
instead of ternary operator, prefer `newBucketMD.capabilities.VeeamSOSApi && { .... }` syntax

```suggestion
                    capabilities: newBucketMD.capabilities && {
                        ...newBucketMD.capabilities,
                        VeeamSOSApi: newBucketMD.capabilities.VeeamSOSApi && {
                            ...newBucketMD.capabilities.VeeamSOSApi,
                            CapacityInfo: newBucketMD.capabilities.VeeamSOSApi.CapacityInfo && {
                                Capacity: new Long(newBucketMD.capabilities.VeeamSOSApi.CapacityInfo.Capacity || '0'),
                                Available: new Long(newBucketMD.capabilities.VeeamSOSApi.CapacityInfo.Available || '0'),
                                Used: new Long(newBucketMD.capabilities.VeeamSOSApi.CapacityInfo.Used || '0'),
                                LastModified: (newBucketMD.capabilities.VeeamSOSApi.CapacityInfo as any).LastModified,
                            },
                        },
                    },
```